Intel_SighPP_CMake/Intel_SighPP_CMake/src/Intel_Sight_++.cpp

```diff
@@ -141,6 +145,54 @@ int main(int argc, char** argv)
 			{
 				SPDLOG_ERROR("Missing value for flag -port");
 			}
+
+			if (next_arg.compare("-caffe") == 0 && (i + 1) < argc)
+			{
+				try
+				{
+					std::string file_ = argv[++i];
+					std::string prototxt_path_ = "./models/" + file_ + ".prototxt";
+					std::string caffemodel_path_ = "./models/" + file_ + ".caffemodel";
+					std::string txt_path_ = "./models/" + file_ + ".txt";
+					CaffeModelImpl caffe_model(prototxt_path_, caffemodel_path_, txt_path_);
+					caffe_models.push_back(caffe_model); 
+					SPDLOG_INFO("Caffe-based network loaded:  {}", file_);
+					continue;
+				}
+				catch (const std::exception& exception)
+				{
+					SPDLOG_CRITICAL("Error with loading caffe-based network from file: {}", exception.what());
+				}
+			}
+			else if (next_arg.compare("-caffe") == 0 && !((i + 1) < argc))
+			{
+				SPDLOG_ERROR("Missing files for caffe-based network");
+				continue;
+			}
+						
+			if (next_arg.compare("-yolo") == 0 && (i + 1) < argc)
+			{
+				try
+				{
+					std::string file_ = argv[++i];
+					std::string cfg_path_ = "./models/" + file_ + ".cfg";
+					std::string weights_path_ = "./models/" + file_ + ".weights";
+					std::string label_path_ = "./models/" + file_ + ".txt";
+					YoloModelImpl yolo_model(cfg_path_, weights_path_, label_path_);
+					yolo_models.push_back(yolo_model);
+					SPDLOG_INFO("Yolo network loaded:  {}", file_);
+					continue;
+				}
+				catch (const std::exception& exception)
+				{
+					SPDLOG_CRITICAL("Error with loading darknet-based yolo network from file: {}", exception.what());
+				}
+			}
+			else if (next_arg.compare("-yolo") == 0 && !((i + 1) < argc))
+			{
+				SPDLOG_ERROR("Missing files for darknet-based yolo network");
```

Comment:
@XingdaCen Are the files missing? or is the flag/argument missing? Same for the -caffe one 😄 ,